src/sentry_plugins/slack/plugin.py

```diff
@@ -17,6 +17,14 @@
 }
 
 
+def track_response_code(status_code):
+    metrics.incr(
+        "sentry-plugins.slack.http_response",
+        sample_rate=1.0,
+        tags={"status": status_code, "plugin": "slack"},
```

Comment:
If slack is in the metric name do you need it in the tag as well? Tags are most useful when you have a common metric name and need to slice it by different dimensions.

Reply:
Can we make this emit the same metric as [here](https://github.com/getsentry/sentry/blob/master/src/sentry_plugins/client.py#L113)?

Reply:
woops, that was leftover from me thinking it'd need to be different because of the OK:true / false business.